agent/api/container/container.go

```diff
@@ -47,6 +47,13 @@ const (
 
 	// AuthTypeASM is to use image pull auth over AWS Secrets Manager
 	AuthTypeASM = "asm"
+
+	// ContainerMetadataURIEnvironmentVariableName defines the name of the environment
+	// variable in containers' config, which can be used by the containers to access the
+	// v3 metadata endpoint
+	ContainerMetadataURIEnvironmentVariableName = "AWS_CONTAINER_METADATA_URI"
```

Comment:
Will `ContainerMetadataURIEnvironmentVariableName` and `ContainerMetadataURIFormat` be used outside of this package?

Reply:
yes it's used in TestCreateContainerV3EndpointIDGenerateSucceeds in task engine unit test